tests/devapps/WAM/UWPWam/MainPage.xaml.cs

```diff
@@ -17,6 +17,7 @@
 // The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
 namespace UWP_standalone
 {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
This is best applied on a single line of  code, not on entire class.

Reply:
This is dead code, will remove